module/modmanager/manager.go

```diff
@@ -849,3 +928,16 @@ func DepsToNames(deps resource.Dependencies) []string {
 	}
 	return depStrings
 }
+
+func getModuleDataParentDirectory(options modmanageroptions.Options) string {
+	// if the home directory is empty, this is probably being run from an unrelated test
+	// and creating an file could lead to race conditions
```

Comment:
`creating a file`

Reply:
I really should get my editor setup to warn me about grammar mistakes... :pray: